lib/aquatone/collectors/crtsh.rb

```diff
@@ -0,0 +1,19 @@
+module Aquatone
+  module Collectors
+    class Crtsh < Aquatone::Collector
+      self.meta = {
+        :name         => "crtsh",
+        :author       => "Joel (@jolle)",
+        :description  => "Uses crt.sh by COMODO CA to find hostnames"
+      }
+
+      def run
+        response = get_request("https://crt.sh/?dNSName=%25.#{url_escape(domain.name)}")
+
+        response.body.to_enum(:scan, /<TD>([a-zA-Z0-9\*_.-]+\.#{domain.name})<\/TD>/).map do |column|
```

Comment:
`Regexp.escape` could be used on `domain.name` to make sure no characters like dots, etc. get treated as part of the regular expression.